pkg/minikube/machine/fix.go

```diff
@@ -94,6 +99,22 @@ func fixHost(api libmachine.API, cc *config.ClusterConfig, n *config.Node) (*hos
 	return h, nil
 }
 
+// warnAboutMisconfiguredMachineConfigPaths will warn users if paths in machine/config.json are misconfigured.
+// This problem may be caused if the minikube home folder is moved to another location https://github.com/kubernetes/minikube/issues/14466
+func warnAboutMisconfiguredMachineConfigPaths(h *host.Host) {
+	var driver map[string]interface{}
+	err := json.Unmarshal(h.RawDriver, &driver)
+	if err != nil {
+		return
```

Comment:
We should do a `klog.Warning` here so we can see in the logs if the driver is failing to unmarshal.

Reply:
Done.